public/content/en/basics/functions.md

```diff
@@ -20,7 +18,7 @@ aware of that.
 Functions might even be declared inside others functions where they may be
 used locally and aren't visible to the outside world.
 These function can even have access to objects local to
-the parent's scope
```

Comment:
Added "Such nested functions are called delegates and will be explained in more depth soon."